forecast-hub-report.Rmd

```diff
@@ -273,13 +274,65 @@ for (location in locations) {
 }
 ```
 
+# Overiew of Scores by Horizon
+
+Overview of all models and metrics stratified by forecast horizons
+
+``` {r scoretablehorizon, fig.width = 12, results = 'asis', eval = TRUE}
+# overall summary
+plot_score_table_helper <- function(data, type, filter) {
```

Comment:
function is redefined here - but also not using the filter argument

Reply:
its all copy and pasted - I'd suggest a clean up and moving the functions outside of the Rmd.